docs/_advanced/lazy_listener.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Lazy listeners (FaaS)
+lang: en
+slug: lazy-listeners
+order: 9
+---
+
+<div class="section-content">
+⚠️ Lazy listener functions are a beta feature to make it easier to deploy Bolt for Python apps to FaaS environments. As the feature is developed, Bolt for Python's API is subject to change.
+
+Typically you'd call `ack()` as the first step of your listener functions. Calling `ack()` tells Slack that you've received the event and are handling it in within reasonable amount of time (3 seconds).
+
+However, apps running on FaaS or similar runtimes that don't allow you to run threads or processes after returning an HTTP response cannot follow this pattern. Instead, you should set the `process_before_response` flag to `true`. This allows you to call ack() and handle the event in seperate threads or processes. For events that do not require acknowledgement, you can create the listener as you normally would.
+
+Rather than acting as a decorator, lazy listeners take two keyword args:
```

Comment:
Then, we can mention why you should use `lazy` in the case. The original one says `To deal with this possible timeout issue, you can use `ack` and `lazy` functions for a listener.` but something similar is fine.

Reply:
Is this accurate now?

Reply:
Thanks, it looks almost done. I added a suggestion above.